Makefile

```diff
@@ -0,0 +1,79 @@
+IMAGE_REGISTRY ?= quay.io/kubevirt
+IMAGE_NAME ?= macvtap-cni
+IMAGE_TAG ?= latest
+
+TARGETS = \
+	goimports-format \
+	goimports-check \
+	whitespace-format \
+	whitespace-check
+
+# Make does not offer a recursive wildcard function, so here's one:
+rwildcard=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2))
+
+# Gather needed source files and directories to create target dependencies
+directories := $(filter-out ./ ./vendor/ ,$(sort $(dir $(wildcard ./*/))))
+all_sources=$(call rwildcard,$(directories),*) $(filter-out $(TARGETS), $(wildcard *))
+
+all: format check
+
+check: goimports-check whitespace-check test/unit
+
+format: goimports-format whitespace-format
```

Comment:
In addition to a missing build target, also missing the typical targets for gofmt and govet. Should these also be run within a builder image?

Reply:
We can add these ourselves later, on another PR.

Let's move this forward.